src/averageAges.js

```diff
@@ -56,6 +93,33 @@ function calculateWomenAverageAge(people, withChildren) {
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
   // write code here
+  const womenOnly = people.filter(({
+    sex,
+  }) => sex === 'f');
+  const mothers = womenOnly.filter(
+    woman => people.find(person => woman.name === person.mother)
+  );
+  let children = people.filter(
+    person => mothers.find(mother => person.mother === mother.name)
+  );
```

Comment:
new line before each new variable declaration

```suggestion
  const womenOnly = people.filter(({ sex }) => sex === 'f');
 
   const mothers = womenOnly.filter(
    woman => people.find(person => woman.name === person.mother)
  );
  
  let children = people.filter(
    person => mothers.find(mother => person.mother === mother.name)
  );
```